GlucoseOnFhir/Float+Extension.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  GlucoseHelper.swift
+//  GlucoseOnFhir
+//
+//  Created by Kevin Tallevi on 7/14/17.
+//  Copyright © 2017 eHealth Innovation. All rights reserved.
+//
+
+import Foundation
+
+extension Float {
+    func truncateMeasurement() -> Float {
+        let formatter = NumberFormatter()
+        formatter.maximumFractionDigits = 2
+        formatter.roundingMode = NumberFormatter.RoundingMode.down
+        let truncatedValue = formatter.string(from: NSNumber(value: self))
+        
+        return Float(truncatedValue!)!
+    }
+}
```

Comment:
A few unit tests around this might be good insurance that this works as expected.